torch/nn/modules/transformer.py

```diff
@@ -174,22 +191,28 @@ def __init__(self, decoder_layer, num_layers, norm=None):
         self.num_layers = num_layers
         self.norm = norm
 
-    def forward(self, tgt, memory, tgt_mask=None, memory_mask=None):
+    def forward(self, tgt, memory, mask=None,
```

Comment:
Similarly, no changes to the APIs. Backward capability issue. See the comment below.